Resources/Prototypes/Traits/species.yml

```
@@ -66,6 +68,45 @@
+  functions:
+    - !type:TraitReplaceComponent
+      components:
+      - type: OniDamageModifier
+        modifiers:
+          coefficients:
+            Blunt: 1.1
+            Slash: 1.1
+            Piercing: 1.1
+      - type: PotentiaModifier
+        potentiaMultiplier: 1.15
+  requirements:
```

YAMLlint (1.35.1)

[error] duplication of key "requirements" in mapping

(key-duplicates)